app/controllers/admins/public_servants_controller.rb

```diff
@@ -121,12 +121,13 @@ def record_number_options
 
   def search_public_servants
     if params[:q].present?
-      @public_servants = @public_servants.where(name: params[:q][:name] ) unless params[:q][:name].blank?
+      @public_servants = @public_servants
+                             .where(id: params[:q][:id]) unless params[:q][:id].blank?
```

Comment:
Line is too long. [86/80]<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.